### ISSUE TITLE
Implicit relative imports break python3 support. 

### ISSUE THREAD

**Issue by bartekbrak** [redouane/django-fontawesome#27]
Import mechanism changed in python3 and implicit relative imports won't work any more. 
```
from forms import IconFormField
ImportError: No module named 'forms'
```

`from forms import IconFormField` needs to become `from .forms import IconFormField`, similar in other places.

This is a trivial change and I will send a PR very soon. The change is backwards compatible, you can read more about it [here](http://stackoverflow.com/questions/22493529/no-module-named-forms-django).

**Comment by bartekbrak** [comment]
Seems like the issue was fixed in [PR#6](https://github.com/redouane/django-fontawesome/pull/6) and [PR#3](https://github.com/redouane/django-fontawesome/pull/3). They are both correct although their commit mesages could be improved (see mine attached below). 

@redouane Why are they not merged? Can I help with this somehow?

I propose this commit message
```
    Change implicit relative imports to explicit
    
    to make them compatible with python3. PEP-404 changed the import
    mechanism so that you have to state that you're importing from the same
    directory.
    
    This change is backwards compatible with python2.
```

**Comment by redouane** [comment]
fixed / merged
